iceoryx_examples/iceoptions/README.md

```diff
@@ -76,12 +87,12 @@ subscriberOptions.queueCapacity = 10U;
 ```
 
 `historyRequest` will enable a subscriber to receive the last n samples on subscription e.g. in case it was started later than the publisher.
-If the publisher does not have a sufficient `historyCapacity` (larger than `historyRequest`), it will still be connected but we will not be able to
+If the publisher does not have a sufficient `historyCapacity` (smaller than `historyRequest`), it will still be connected but we will not be able to
```

Comment:
It is probably a little ambiguous here: sufficient iff hC >= hR (larger, or more precisely at least as large ...).

Hence: not sufficient iif hC < hR, as stated here.